src/keyring/KeyringController.ts

```diff
@@ -421,6 +428,25 @@ export class KeyringController extends BaseController<
   ) {
     try {
       const address = normalizeAddress(messageParams.from);
+      const QRKeyring = await this.getOrAddQRKeyring();
```

Comment:
Should this be `qrKeyring` instead of `QRKeyring`? Otherwise we may confuse this for a class of some sort.